services/py-api/src/service/hackathon_service.py

```diff
@@ -65,6 +65,24 @@ async def create_random_participant(
         # As when first created, the random participant is not assigned to a team we return the team as None
         return Ok((result.ok_value, None))
 
+    async def verify_admin_participant_and_team_in_transaction(self, admin_id: str) -> Any:
+        return await self._tx_manager.with_transaction(self.verify_admin_participant_and_team_callback, admin_id)
+
+    async def verify_admin_participant_and_team_callback(
+        self, admin_id: str, session: Optional[AsyncIOMotorClientSession] = None
+    ) -> Any:
+        result_admin = await self._participant_repo.update(
+            obj_id=admin_id, input_data={"email_verified": True}, session=session
+        )
+        if is_err(result_admin):
+            return result_admin
+
+        result_team = await self._team_repo.update(obj_id=admin_id, input_data={"is_verified": True}, session=session)
+        if is_err(result_team):
+            return result_admin
+
+        return Ok(value=True)
```

Comment:
Why are you returning boolean here? Maybe return the updated Team and Participant documents.